config/packages/workflow.yaml

```diff
@@ -0,0 +1,2 @@
+framework:
+    workflows: null
```

Comment:
Is it required to be placed in all packages that will be using this feature? If we will keep the workflow as a hard dependency, should we keep this config in part of a package that will be loaded?

Reply:
You're right, we should enable it on Core bundle.

Reply:
I added this configuration on Core bundle too. Have I to remove it here?

Reply:
yes please, to see if it work without any change in config in end apps